inference-engine/src/vpu/graph_transformer/include/vpu/model/stage.hpp

```diff
@@ -167,6 +167,7 @@ VPU_DECLARE_ENUM(StageType,
     OutShapeOfReshape = 127,
     Concat = 128,
     Broadcast = 129,
+    LoopDynamicIterNumber = 130,
```

Comment:
Is there conflict with `StaticShapeNMS`? @AndrewBakalinIntel could you please take a look?

Reply:
Nitpick: `LoopDynamicIterNumber` -> `LoopIterationsCount`

Reply:
Fixed